flavors/etcdkafka/etcd_kafka_flavor.go

```diff
@@ -44,6 +45,11 @@ func (f *Flavor) Inject() error {
 	f.Etcd.LogFactory = &f.Generic.Logrus
 	f.Etcd.ServiceLabel = &f.Generic.ServiceLabel
 	f.Etcd.StatusCheck = &f.Generic.StatusCheck
+
+	f.EtcdSync.Aggregator = &f.Generic.Transports
```

Comment:
transport has to be added to aggregator somehow, so I removed it from etcd plugin and added here